### ISSUE TITLE
Fix Babel 6.4: A semicolon is required after a class property

### ISSUE THREAD

**Issue by ChrisCinelli** [AtomLinter/linter-jscs#171]
I upgraded Babel to 6.4. Before semicolon after class property were not mandatory but now I have ton of :

    Module build failed: SyntaxError: 
    ...
    A semicolon is required after a class property

Do you know of a preset to automatically fix these automatically? I tried airbn and google without success.

See also http://stackoverflow.com/questions/35103415/autofix-babel-6-4-a-semicolon-is-required-after-a-class-property

**Comment by weslleyaraujo** [comment]
I am having the same issue,

```javascript
// .babelrc
{
  "presets": ["es2015", "react", "stage-0"]
}

// package.json
"babel-core": "^6.0.20",
"babel-loader": "^6.2.0",
"babel-preset-es2015": "^6.1.18",
"babel-preset-react": "^6.1.18",
"babel-preset-stage-0": "^6.3.13",
```

**Comment by electerious** [comment]
Has nothing to do with linter-jscs. However, it looks like a auto-fix as already been merged into jscs: https://github.com/jscs-dev/node-jscs/pull/2057

**Comment by ChrisCinelli** [comment]
My understanding is that node-jscs is packaged with AtomLinter/linter-jscs so it will take a new release to be able to use this feature in AtomLinter/linter-jscs

**Comment by electerious** [comment]
Wasn't aware that node-jscs is packaged with linter-jscs. Would explain why it the new `requireCapitalizedConstructorsNew` property isn't working, yet.

**Comment by Arcanemagus** [comment]
Correct, this plugin uses it's bundled version of `jscs` (for now), it looks like @ChrisCinelli  filed #172 and @casesandberg is taking care of it though :wink:.

**Comment by casesandberg** [comment]
This was fixed in `linter-jscs@3.4.2`
